Mathlib/Probability/Martingale/Upcrossing.lean

```diff
@@ -695,7 +695,7 @@ This inequality is central for the martingale convergence theorem as it provides
 for the upcrossings.
 
 We note that on top of taking the supremum on both sides of the inequality, we had also used
-the monotone convergence theorem on the left hand side to take the supremum outside of the
+the monotone convergence theorem on the left-hand side to take the supremum outside of the
```

Comment:
I disagree with this one, please revert.

Reply:
With the hyphen looks more correct to my eye, and I guess the general rule is that you hyphenate a compound adjective when it comes before the noun it modifies: https://en.wikipedia.org/wiki/Compound_modifier#Hyphenation_of_elements_in_English

It'd be nice if we could be consistent, but I don't feel too strongly about this; either way it will be understood and hopefully not be too jarring.

Reply:
My reasoning was the same as yours @bryangingechen. If this proves too contentious, I can split this into a separate PR, but, in all honesty, my impression as of now is that "left-hand side" is strictly more correct than "left hand side".